client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -324,6 +324,11 @@ export function getEnhancedTasks(
 						badgeText: isPaidPlan ? '' : translate( 'Upgrade plan' ),
 					};
 					break;
+				case 'verify_email':
+					taskData = {
+						title: translate( 'Verify Email (Check Your Inbox)' ),
```

Comment:
Would be great to get feedback from @nuriapenya about the text that should appear here and if we need anything more than this.

Reply:
@edanzer Upon checking the actual email users receive (here you [can see the template](https://mc.a8c.com/email-previewer/?theme=wpcom-2022&template=activation&locale=en&variables%5Bcta_url%5D=https%3A%2F%2Fwordpress.com%2F&variables%5Bcta_label%5D=Click+here)), I would suggest changing to the following copy:

`Confirm email (check your inbox)`

Reply:
@nuriapenya apologies for the delay in getting to this. I think your suggestion above makes perfect sense.